__tests__/contacts/contacts.mutation.test.js

```diff
@@ -0,0 +1,380 @@
+                statusCode
+                message
+              }
+            }
+          }
+        `,
+        variables: { contact },
+      })
+      .then(response => response)
+      .catch(e => e);
+    expect(res.data.addContact).toBeDefined();
+    expect(res.data.addContact).toHaveProperty(
+      'message',
+      CONTACT_ALREADY_EXIST,
+    );
+    expect(res.data.addContact).toHaveProperty('statusCode', 400);
+  });
+
+  test('#3 update contact', async () => {
+    const res = await client
+      .mutate({
+        mutation: gql`
+          mutation($id: ID!, $contact: contactInput!) {
+            updateContact(id: $id, contact: $contact) {
+              ... on Contact {
+                _id
+                phoneNumber
+                openHours {
+                  lang
+                  value
+                }
+                address {
+                  lang
+                  value
+                }
+                email
+                images {
+                  medium
+                }
+                link
+              }
+              ... on Error {
+                message
+                statusCode
+              }
+            }
+          }
+        `,
+        variables: { id: contactsId, contact: updatedContact },
+      })
+      .catch(e => e);
+    expect(res.data.updateContact).toHaveProperty('email', 'test@test.com');
+    expect(res.data.updateContact.images).toBeInstanceOf(Object);
+    expect(res.data.updateContact).toHaveProperty('images', {
+      __typename: 'ImageSet',
+      medium: null,
+    });
+    expect(res.data.updateContact).toHaveProperty('address', [
+      { __typename: 'Language', lang: 'uk', value: 'Вулиця 3' },
+      { __typename: 'Language', lang: 'en', value: 'Street 3' },
+    ]);
+  });
+
+  test('#4 update not existing contact should return error', async () => {
+    const res = await client
+      .mutate({
+        mutation: gql`
+          mutation($id: ID!, $contact: contactInput!) {
+            updateContact(id: $id, contact: $contact) {
+              ... on Contact {
+                _id
+                phoneNumber
+                openHours {
+                  lang
+                  value
+                }
+                address {
+                  lang
+                  value
+                }
+                email
+                images {
+                  medium
+                }
+                link
+              }
+              ... on Error {
+                message
+                statusCode
+              }
+            }
+          }
+        `,
+        variables: { id: notExistContactId, contact: updatedContact },
+      })
+      .then(response => response)
+      .catch(e => e);
+    expect(res.data.updateContact).toHaveProperty('message', CONTACT_NOT_FOUND);
+    expect(res.data.updateContact).toHaveProperty('statusCode', 404);
+  });
+
+  test('#5 update not existing contact should return error', async () => {
```

Comment:
I think this test is almost complete duplication of the test on the line `#223`. Should they be so similar to each other?